src/component/contentpost/index.tsx

```diff
@@ -171,12 +172,10 @@ function ContentPost(props: ContentPostProps) {
         },
       }}
       description={{
-        as: ComposeEditor,
+        as: ReadonlyEditor,
```

Comment:
Eventually ReadonlyEditor and ComposeEditor should be combined because they are very similar.